battles/helpers/battle.py

```diff
@@ -0,0 +1,78 @@
+from collections import Counter
+
+from battles.helpers.emails import send_email_when_battle_finishes
+from battles.helpers.fight import compare_two_pokemons
+from battles.models import BattleTeam
+from users.models import User
+
+
+def can_run_battle(battle):
+    try:
+        BattleTeam.objects.get(
+            battle_related=battle, trainer=battle.creator)
+        BattleTeam.objects.get(
+            battle_related=battle, trainer=battle.opponent)
+    except BattleTeam.DoesNotExist:
+        return False
+    else:
+        return True
+
+
+def mount_battle_list(battle):
+    creator_team = BattleTeam.objects.get(
+        battle_related=battle, trainer=battle.creator)
+    opponent_team = BattleTeam.objects.get(
+        battle_related=battle, trainer=battle.opponent)
+    result = {}
```

Comment:
Nothing wrong with doing it this way, but you could also do it in a cleaner way:
```
result = {
    'creator_team': [pokemon.id for pokemon in creator_team.pokemons.all()],
    'opponent_team': [pokemon.id for pokemon in opponent_team.pokemons.all()],
}
```